e2e/runner/update_contracts.go

```diff
@@ -0,0 +1,38 @@
+package runner
```

Comment:
I think we can name the file `admin_evm`

Admin actions related to the evm chain